guides/release/configuring-ember/optional-features.md

```diff
@@ -170,3 +170,45 @@ backwards compatibility for existing templates while new template-only
 components gain the advantages of this feature.
 
 For more information, see [RFC #278](https://github.com/emberjs/rfcs/blob/master/text/0278-template-only-components.md).
+
+### default-async-observers
+
+With this feature *enabled*, Ember will run all observers in the application
+asynchronously by default. This leads to observers running in the runloop
+after the one in which the properties it observers were updated.
+
+If the feature is *disabled*, observers run synchronously
+and will then update as soon as their observed properties update.
+
+Async observers are more performant than those that run synchronously
+and can help you to manage your application state in a more predictable manner.
+This is one of the reasons, why the `default-async-observers` feature is
+**enabled by default** in newly created, modern Ember applications.
+
+The `default-async-observers` feature affects the behaviour of observers application-wide,
+but can still set single instances of observers to run synchronously or async
+manually. By using the `sync: true` option, observers who are otherwise async by default,
+can be marked as synchronous manually. Vice versa, otherwise synchronously running observers
+can be set to run asynchronously using the `sync: false` option.
+
+
+```javascript
+import { observer } from '@ember/object';
+
+Person.reopen({
+  partOfNameChanged: observer({
+  dependentKeys: ['firstName', 'lastName'],
+  function() {
+    // Fires async after firstName or lastName have updated
+  },
+  sync: false,
+});
+```
+
+While the `default-async-observers` feature is only enabled by default in modern Ember applications,
+you can enable this optional feature in older apps (Ember 3.13+) as follows:
```

Comment:
Let me know if you find it more useful to just skip this part of the phrase referring to the old behaviour. If we skip this part, we could also just show how to disable the feature in current Ember apps in the code example below